### ISSUE TITLE
No longer works too many requests

### ISSUE THREAD

**Issue by munntjlx** [flavienbwk/Regex101-Offline#1]
tried this and it seems to need some kind of delay.

**Comment by flavienbwk** [comment]
Hi, please try with the `flavienb/regex101-offline:latest` image, without building (comment the `build: ` line in docker-compose.yml).

**Comment by munntjlx** [comment]
Agree, but the image has lots of scan issues being that it is 4 years old.

________________________________________

**Comment by flavienbwk** [comment]
This repo uses the official Python 3.7 image. You might want to use 3.11, there are no security issues to fix apart from that (or favor a dedicated Dockerfile that suits your security needs).

I've [tried rebuilding it with the 2024 website](https://github.com/flavienbwk/Regex101-Offline/tree/2024-upgrade) but it looks like regex101.com has some kind of new security measures and the `wget` technique doesn't work anymore (connection refused). You might want to use some [advanced technique](https://playwright.dev/) yourself (that requires time to develop) to download files (or [hire me](https://berwick.fr/en) to do it).
